### PR TITLE
Reorg for release notes

### PR DESCRIPTION
**Pull request by NWeber-SL**

### CODE REVIEW

File: using-ocular/common-queries/img/. ... [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/common-queries/img/. ...]: Please remove this file

File: using-ocular/common-queries/. ... [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/common-queries/. ...]: Please remove this file

File: SUMMARY.md [hunk 3]
```diff
@@ -17,12 +17,12 @@
...
-    * [Analyzing Types and Packages](using-ocular/use-cases/types-packages-analysis.md)
-    * [Analyzing Method Parameters](using-ocular/use-cases/parameters-analyze.md)
-    * [Analyzing Methods](using-ocular/use-cases/methods-analyze.md)
-    * [Uncovering Data Flows](using-ocular/use-cases/data-flows.md)
+  * Common Queries
+    * [Analyzing Dependencies](using-ocular/common-queries/dependency-analysis.md)
+    * [Analyzing Types and Packages](using-ocular/common-queries/types-packages-analysis.md)
+    * [Analyzing Method Parameters](using-ocular/common-queries/parameters-analyze.md)
+    * [Analyzing Methods](using-ocular/common-queries/methods-analyze.md)
+    * [Uncovering Data Flows](using-ocular/common-queries/data-flows.md)
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: What prompts the change of this so frequently? Every time we change this, our links no longer work. If users have shared these pages amongst them, the links become unusable.

Comment (NWeber-SL) [reply SUMMARY.md]: I'm restructuring the Ocular docs as more articles are added and as I'm learning more about the product. These aren't really use cases, instead explaining common queries for things users might want to do. And we don't want these articles in the use cases bucket anyway as that will be getting bigger. Sorry, should have discussed with you first.

Comment (vlad-alexandru-ionescu) [reply SUMMARY.md]: Ok

File: using-ocular/common-queries/img/. ... (hunk 1 above)
Comment (NWeber-SL) [reply using-ocular/common-queries/img/. ...]: Done

File: using-ocular/common-queries/. ... (hunk 2 above)
Comment (NWeber-SL) [reply using-ocular/common-queries/. ...]: Done

**Pull request merged**